docker-compose.yaml

```diff
@@ -0,0 +1,32 @@
+services:
+  cinema:
+    build:
+      context: .
+    env_file:
+      - .env
+    ports:
+      - "8001:8000"
+    command: >
+      sh -c "
+      python manage.py makemigrations &&
+      python manage.py migrate &&
+      python manage.py runserver 0.0.0.0:8000
```

Comment:
Consider using a process manager like `gunicorn` for running the Django application in a production environment instead of `runserver`, which is not suitable for production use.